src/jit/importer.cpp

```diff
@@ -9763,6 +9763,99 @@ var_types Compiler::impGetByRefResultType(genTreeOps oper, bool fUnsigned, GenTr
 }
 
 //------------------------------------------------------------------------
+// impOptimizeCastClassOrIsInst: attempt to resolve a cast when jitting
+//
+// Arguments:
+//   op1 -- value to cast
```

Comment:
```
// Arguments:
//    <argument1-name> - Description of argument 1
//    <argument2-name> - Description of argument 2
```

Reply:
Fixed (though I happen to like `--` better).